main.go

```diff
@@ -67,7 +67,7 @@ func (s *service) start() error {
 	listener, err := net.Listen("tcp", s.config.Listen)
 
 	if err != nil {
-		return err
+		panic(err)
```

Comment:
Change to `Fatal` here [main.go:48] (https://github.com/allegro/akubra/blob/master/main.go#L48) instead.